automation/Dockerfile-tests

```diff
@@ -4,6 +4,7 @@ COPY src /app/src
 COPY test.sh /app
 COPY project /app/project
 COPY build.sbt /app
+COPY logback-test.xml /app
```

Comment:
@jacmrob Hi Jackie, Is this change okay?

Reply:
have you tested that this stuff works in docker?  (that the code is looking for it at the correct path?) it will be copied into the /app directory of the automation docker container